package.sh

```diff
@@ -22,11 +22,11 @@ debian_package(){
     echo "Section: custom"
     echo "Priority: optional"
     echo "Architecture: all"
-    echo "Depends: util-linux"
+    echo "Depends: util-linux, python3 (>= 3.7), python3-sdnotify, python3-posix-ipc"
     echo "Essential: no"
-    echo "Installed-Size: 16"
+    echo "Installed-Size: $(du -s "$DEB_NAME" | cut -f1)"
     echo "Maintainer: nefelim4ag@gmail.com"
-    echo "Description: Script for creating hybrid swap space from zram swaps, swap files and swap partitions."
+    echo "Description: Script for creating hybrid swap space from zram swaps, swap files, and swap partitions."
```

Comment:
Afaik there shouldn't be a comma here

Reply:
See https://github.com/Nefelim4ag/systemd-swap/pull/164#discussion_r456914847.